homework25/pages/base_page.py

```diff
@@ -0,0 +1,61 @@
+"""Module for defining the BasePage class"""
+
+
+class BasePage:
+    """
+    Base class for page objects.
+
+    Provides common functionality for interacting with web pages, such as:
+        - Initializing page objects with a driver and URL
+        - Opening URLs
+        - Finding elements
+    """
+
+    def __init__(self, driver, url, timeout=10):
+        """
+        Initializes a BasePage object.
+
+        Args:
+            driver (webdriver): WebDriver instance.
+            url (str): Base URL of the page.
+            timeout (int, optional): Implicit wait timeout in seconds.
+            Defaults to 10.
+        """
+        self.driver = driver
+        self.url = url
+        self.driver.implicitly_wait(timeout)
```

Comment:
you have already use it in `conftest` file

Reply:
Removed duplicate lines from conftest file